app/helpers/vacancies_options_helper.rb

```diff
@@ -0,0 +1,42 @@
+module VacanciesOptionsHelper
+  def job_location_options
```

Comment:
Mild preference for these to be in alphabetical order for easy finding, if all else is equal

Reply:
Will do that now sir!